clair/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: postgresql
+  repository: https://kubernetes-charts.storage.googleapis.com
```

Comment:
Use our postgresql chart. There are tweaks to make it work better with our cluster.